### PR TITLE
Add a bunch of publications.

### PR DESCRIPTION
**Pull request by marcfehling**

### CODE REVIEW

**Review by tjhei — APPROVED** [review]

File: publications-2022.bib [hunk 1]
```diff
@@ -1156,6 +1171,16 @@ @Article{2022:schussnig.pacheco.ea:semi-implicit
   publisher = {Elsevier {BV}}
 }
 
+@PhDThesis{2022:schussnig:generalised,
+  author  = {Schussnig, Richard},
```
Comment (peterrum) [inline publications-2022.bib:1175]: @richardschu FYI

File: publications-2023.bib [hunk 2]
```diff
@@ -233,6 +306,19 @@ @Article{2023:vu.rodriguez.ea:high-accuracy
   preprint = {https://arxiv.org/abs/2111.06893}
 }
 
+@Article{2023:weerdesteijn.naliboff.ea:modeling,
```
[the remark this answers is not in the document]
Comment (marcfehling) [reply publications-2023.bib:309]: I signed up for a personal researchgate newsletter, and quickly translate announced papers to the publication list :)

**Review by peterrum — APPROVED** [review]
Thx!

File: publications-2022.bib (hunk 1 above)
Comment (richardschu) [reply publications-2022.bib:1175]: #poweredbydeal.II

**Pull request merged**